controller/validations.go

```diff
@@ -32,8 +32,8 @@ func NewValidationsController(
 	mgr ctrl.Manager,
 ) error {
 	reconcileInterval := conf.ExternalServices.Istio.ValidationReconcileInterval
-	log.Infof("Kiali will validate Istio configuration every: %s", *reconcileInterval)
 	reconciler := NewValidationsReconciler(clusters, conf, kialiCache, validationsService, *reconcileInterval)
+	reconciler.log.Infof("Kiali will validate Istio configuration every: %s", *reconcileInterval)
```

Comment:
The manager has its own logger that needs to be set otherwise some errors are thrown. Typically reconcilers inherit from that logger but I don't think you have to. Not sure if it adds any additional context or if its just a convenience.

Reply:
OK, I just learned a lesson here. Don't assume the context.Context getting passed around is the same context everywhere. The manager has a context.Context but the Reconciler, which also has a context, is not the same context as that of the parent manager. I don't know where that context comes from but it isn't from the manager and thus I can't flow the logger from manager to reconciler. I fixed this in the latest commit.

Reply:
> I don't use the one in the manager because that now uses yet another logging API (logr.Logger) - I'm loathe to start using that now

This is actually a good example of how the backend logger can stay zerolog while the interface is different since we're now passing `zerolog-logr`.

> Don't assume the context.Context getting passed around is the same context everywhere. The parent controller has a context.Context but the Reconciler, which also has a context, is not the same context as that of the parent controller. 

Context matters... The context passed to `Start` is the same one passed to all the startup components from `kiali.go`. When this gets cancelled the manager stops. Whereas the context in Reconcile is probably generated each time Reconcile is called.